tests/quotes.pyi

```diff
@@ -53,3 +53,7 @@ class DocstringAndEllipsis:
 class DocstringAndPass:
     """Docstring"""  # Y021 Docstrings should not be included in stubs
     pass  # Y012 Class body must not contain "pass"
+
+# These two shouldn't trigger Y020 -- empty strings can't be "quoted annotations"
+k = u""  # Y015 Bad default value. Use "k = ..." instead of "k = u''"
+el = r""  # Y015 Bad default value. Use "el = ..." instead of "el = ''"
```

Comment:
```suggestion
k = ""  # Y015 Bad default value. Use "k = ..." instead of "k = ''"
el = r""  # Y015 Bad default value. Use "el = ..." instead of "el = ''"

# The following should also pass,
# But we can't test for it in CI, because the error message is *very* slightly different on 3.7
#
# On 3.7:
# m = u""  # Y015 Bad default value. Use "m = ..." instead of "m = ''"
# On 3.8+:
# m = u""  # Y015 Bad default value. Use "m = ..." instead of "m = u''"
```